application/clicommands/ListCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Icinga\Module\Reporting\Clicommands;
+
+use Icinga\Module\Reporting\Cli\Command;
+use ipl\Sql\Select;
+
+class ListCommand extends Command
+{
+    /**
+     * List reports
+     *
+     *  USAGE:
+     *
+     *   icingacli reporting list --orderby id|name|author --sort ASC|DESC (default DESC)
+     */
+
+    public function indexAction()
+    {
+        $orderby = $this->params->get('orderby', 'r.mtime');
+        $sortDir = $this->params->get('sort', 'DESC');
+
+        $select = (new Select())
+            ->from('report r')
+            ->columns(['r.*', 'timeframe' => 't.name', 'class' => 'rl.class'])
+            ->join('timeframe t', 'r.timeframe_id = t.id')
+            ->join('reportlet rl', 'r.id = rl.report_id')
+            ->orderBy($orderby, $sortDir);
+
+        $reports = $this->getDb()->select($select);
+
+        foreach ($reports as $r) {
+                $className = (new $r->class())->getName();
+                echo "Id: $r->id  \t Name: $r->name \t Author: $r->author \t Type:  $className \n";
+        }
```

Comment:
I think it looks slightly better when you do it that way:
```suggestion
        $format = "|%5.5s | %-30.30s | %-30.30s | %-40.40s |\n";
        $beautifier = sprintf($format, str_repeat('-', 5), str_repeat('-', 30), str_repeat('-', 30), str_repeat('-', 40));
        printf($format, 'Id', 'Name', 'Author', 'Type');
        printf($beautifier);

        foreach ($reports as $r) {
            $className = (new $r->class())->getName();
            printf($format, $r->id, $r->name, $r->author, $className);
        }

        printf($beautifier);
```